src/v2/layouts/MemberDataGrid/MemberDataGridContainer.tsx

```diff
@@ -38,12 +38,13 @@ export const MemberDataGridContainer = (): JSX.Element => {
 	const [checkIds, setCheckIds] = useState<StudentCorrectionResponse[]>([]);
 	const [isVisibleConfigModal, setIsVisibleConfigModal] = useState<boolean>(false);
 	const { headerConfig } = memberGridMetaData;
+	const [activeStatus, setActiveStatus] = useState('Active');
```

Comment:
Create a constant map for status values and then use it here
```suggestion
	const [activeStatus, setActiveStatus] = useState(STATUS.Active);
```